packages/storage/src/bucket.js

```diff
@@ -1428,6 +1428,46 @@ Bucket.prototype.setMetadata = function(metadata, options, callback) {
 };
 
 /**
+ * Set the storage class for this bucket.
+ *
+ * @resource [Storage Classes]{@link https://cloud.google.com/storage/docs/storage-classes}
+ *
+ * @param {string} storageClass - The new storage class. (`multi_regional`,
```

Comment:
Pulling in from documentation: Values include MULTI_REGIONAL, REGIONAL, STANDARD, NEARLINE, COLDLINE, and DURABLE_REDUCED_AVAILABILITY.